actions/pack_action.go

```diff
@@ -14,9 +14,10 @@ Mandatory properties:
 
 Optional properties:
 
-- compression -- compression type to use. Currently only 'gz', 'bzip2' and 'xz'
-compression types are supported. Use 'none' for uncompressed tarball. The 'gz'
-compression type will be used by default.
+- compression -- compression type to use. Currently only 'gz', 'bzip2', 'xz' and 'pigz'
```

Comment:
you're confusing this - this documentation snippet should be the same as before as nothing user-facing has changed :-).